graphqlapi/src/main/java/io/stargate/graphql/schema/cqlfirst/dml/fetchers/DmlFetcher.java

```diff
@@ -150,16 +152,11 @@ protected Parameters getDatastoreParameters(DataFetchingEnvironment environment)
   }
 
   protected String getDBColumnName(Table table, String fieldName) {
-    Column column = getColumn(table, fieldName);
-    if (column == null) {
-      return null;
-    }
-    return column.name();
+    return dbColumnGetter.getDBColumnName(table, fieldName);
```

Comment:
Maybe we can make `dbColumnGetter` protected and inline and remove these methods?